lib/optimizely/event_builder.rb

```diff
@@ -94,9 +94,10 @@ def get_common_params(user_id, attributes)
             visitor_id: user_id
           }
           ],
-          revision: @config.revision,
-          client_name: CLIENT_ENGINE,
-          client_version: VERSION
+        anonymize_ip: @config.should_anonymize_ip,
+        revision: @config.revision,
+        client_name: CLIENT_ENGINE,
+        client_version: VERSION
```

Comment:
plz. indent last 4 lines.

Reply:
They are inline with it's sibling parameters. See above.